api/android/build-android-lib.sh

```diff
@@ -13,39 +13,49 @@
 # - SNAP_DIRECTORY: Absolute path for SNAP, tensor-filter sub-plugin and prebuilt library.
 #
 # Build options
-# --api_option (default 'all', 'lite' to build with GStreamer core plugins)
+# --build_type (default 'all', 'lite' to build with GStreamer core plugins)
 # --target_abi (default 'armv7, arm64')
 # --run_test (default 'no', 'yes' to run the instrumentation test)
-# --enable_snap (default 'no', 'yes' to build with sub-plugin for SNAP)
+# --enable_snap (default 'yes' to build with sub-plugin for SNAP)
+# --enable_tflite (default 'yes' to build with sub-plugin for tensorflow-lite)
 #
 # For example, to build library with core plugins for arm64-v8a
 # ./build-android-lib.sh --api_option=lite --target_abi=arm64
 #
 
-# API build option ('lite' to build with GStreamer core plugins)
+# API build option
+# 'all' : default
+# 'lite' : with GStreamer core plugins
+# 'single' : no plugins, single-shot only
+# 'internal' : no plugins, single-shot only, disable SNAP and tf-lite
```

Comment:
How about this? `all`, `core`, `single`, `single-internal`

Reply:
Build for internal release : I am not sure internal release includes only single-shot functions. (TBD)
About the 'lite', in the past we determined the lib name as nnstreamer-lite with minimized plugins.